docs_app/content/tutorial/6-chaining-operators.md

```diff
@@ -0,0 +1,14 @@
+# 6. Chaining Operators
+
+## TLDR:
+
+- `pipe` is a functional programming concept.
+- Use `Observable` [`pipe`](API).
```

Comment:
Consider expanding the second bullet point to "Use Observable pipe to chain multiple operators".